### PR TITLE
Use negative prompts in JAX docs

### LINKED ISSUE #4218: diffusers.FlaxStableDiffusionPipeline doesnt have negative prompts
as title 🤷‍♀️🤷‍♀️
diffusers.FlaxStableDiffusionPipeline doesnt have negative prompts so any model checkpoint cant generate any decent faces

### PR DESCRIPTION
**Pull request by pcuenca**
Source notebook to be updated in https://github.com/huggingface/notebooks/pull/416

Fixes #4218

I also removed the `pmap` section because the private `_generate` function has changed a lot, and explanation would be a lot more confusing now.

### CODE REVIEW

**Comment by sayakpaul** [comment]
> I also removed the pmap section because the private _generate function has changed a lot, and explanation would be a lot more confusing now.

Does this have performance implications? If so, I think it would be nice to document them.

File: docs/source/en/using-diffusers/stable_diffusion_jax_how_to.mdx [hunk 1]
```diff
@@ -202,50 +213,3 @@ image_grid(images, 2, 4)
 ```
 
 ![img](https://huggingface.co/datasets/YiYiXu/test-doc-assets/resolve/main/stable_diffusion_jax_how_to_cell_43_output_0.jpeg)
-
-
-## How does parallelization work?
```
Comment (sayakpaul) [inline docs/source/en/using-diffusers/stable_diffusion_jax_how_to.mdx:207]: This was very valuable info no?

**Comment by pcuenca** [comment]
@sayakpaul good point. The reason is that the `_generate` function used to be very easy to understand with `pmap`, but now `pmap` looks like this: https://github.com/huggingface/diffusers/blob/06eda5b232e3bddbcff3281ed1ab18403c243486/src/diffusers/pipelines/stable_diffusion/pipeline_flax_stable_diffusion.py#L434C10-L434C10

We introduced more parameters, some of which can be changed without recompiling. Preparing the inputs (some sharded, some not) appropriate for the call and explaining what all those options to `pmap` mean is harder than the previous example we had.

I can try to do it anyway and we can discuss if it's worth keeping even if it's more complex.

**Comment by sayakpaul** [comment]
If it's not too much let's maybe give it a try? I really like the information that is apparently going away in this PR and I think the community members would feel the same way.

File: docs/source/en/using-diffusers/stable_diffusion_jax_how_to.mdx (hunk 1 above)
Comment (stevhliu) [reply docs/source/en/using-diffusers/stable_diffusion_jax_how_to.mdx:207]: I think it'd be nice to put this section in the Conceptual Guides (you can call it JAX parallelization or something like that), and add a link to it from this doc. To me, this info is more explanatory and good for deepening a user's understanding but not as helpful when they're just looking for the steps on how to do something.

**Review by stevhliu — APPROVED** [review]
Thanks for the cool update! Should we also mention in the [Quicktour](https://huggingface.co/docs/diffusers/v0.18.2/en/quicktour) that you can use `negative_prompts` to get better results?

**Review by patrickvonplaten — APPROVED** [review]

**Review by yiyixuxu — APPROVED** [review]
awesome! thanks

**Comment by patrickvonplaten** [comment]
@pcuenca do you want to go ahead with the PR?

**Pull request closed without merging**